client/luigi-client.d.ts

```diff
@@ -259,6 +259,14 @@ export type getToken = () => AuthData['accessToken'];
 export function getEventData(): Context;
 export type getEventData = () => Context;
 
+/**
+ * Returns the context object. It invokes the getEventData() function.
```

Comment:
Maybe it is more elegant to say here and in the docu that it is an alias of the getEventData function instead of saying that it invokes it?